tests/runtests.py

```diff
@@ -233,6 +234,22 @@ def actual_test_processes(parallel):
         return parallel
 
 
+def validate_selenium(selenium_args):
+    """
+    Validate the browsers requested and raise appropriate exception.
+    """
+    # Remove any browser repition.
+    browsers = list(set(selenium_args.split(',')))
+    for browser in browsers:
+        try:
+            # Create a class instance to see if everything is fine.
+            import_string('selenium.webdriver.%s.webdriver.WebDriver' % browser)().quit()
```

Comment:
Sure.
I have an exam in a few hours, so I am planning to study a bit now :P
Will do this tomorrow.